app/main.py

```diff
@@ -1,3 +1,107 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
other can be `float` and `int`, you should show it in type hint. Fix everywhere, please
```suggestion
    def __add__(self, other: "Distance" | float | int) -> "Distance":
```